documents/LORE.md

```diff
@@ -0,0 +1,25 @@
+# The Stickian Empire
+
+## Topics
+
+1. Many years ago, the rubberian people were defeated at the Great War of the Rubberhand
+2. All realms across the now called Rubberfree land were living in peace, focused on prosperity and growth
+3. The king Madstick, the III, was not happy. Despite being decorated as the strongest general, he didn't receive a great price when the war ended.
+4. He waited for some years until the union between realms was not as strong as before, during the war. 
+5. He went to meet the dark wizard on the Forest of The Thousand Sticks, who knew ancient wizardry. The king knew he would help him as he was in debt to him and his family.
+6. The king asked for an army, an huge army that he could have control on, to conquer all the Rubberfree land. The wizard warned the king about the dangers of such request, but the king was already poison by greed.
+7. The wizard couldn't simply make an army from nowhere. He would need an huge amount of bodies and souls to do his magic. So they went to the rubberian graveyard, where no stickman would ever go.
+8. There, the wizard invoked the rubber warriors from the dead and formed an huge army of undead rubber soldiers that were controlled by the king. His army would grow many times as the he wish.
+9. The king, with his army of undead, then started to invade others territories, pillaging every village, raiding every city. His power was growing but so was his army. The Madstick king has become the Madking.
+10. He was godlike. No other realm could stop him. Things were looking great for him, until the day his greed began to destroy him. 
+11. His army of undead were now too strong even for himself, as the wizard warned. His army saw their leader was weak and so they killed him and corrupted him in a way that he became one of them. The Madking became the undead.
+12. Stickian citizens were afraid, but would not ever give up on fighting these beasts. Will you help them reach victory, or will you fall into misery? The choice is yours.
+
+
+## Complete version
```

Comment:
I would not put a "complete version" just yet, as that could be expanded in chapters and books in the se_lore repo

Reply:
Yes, i will replace it by "Summary" or "Short Version"